x-pack/auditbeat/auditbeat.yml

```diff
@@ -57,6 +57,8 @@ auditbeat.modules:
 
   state.period: 12h
 
+  user.detect_password_changes: false
```

Comment:
If we want to do `off` in the code, but `on` in the config, this would be the place to change it.

Reply:
👍 - I've changed it